composer.json

```
@@ -15,17 +15,21 @@
 		}
 	],
 	"require": {
+		"php": ">=7.2",
 		"abraham/twitteroauth": "^0.7.4"
 	},
 	"require-dev": {
-		"phpunit/phpunit": "^7",
+		"phpunit/phpunit": "^7.5",
 		"wp-coding-standards/wpcs": "^2.2",
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.5.0"
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+		"10up/wpacceptance": "dev-master"
 	},
 	"scripts": {
 		"test": "phpunit",
 		"setup-local-tests": "bash bin/install-wp-tests.sh wordpress_test root root localhost latest",
```

```suggestion
		"setup-local-tests": "bash bin/install-wp-tests.sh wordpress_test root root 127.0.0.1 latest",
```

Seems like we can't use a socket connection here, which it assumes from localhost, so let's try 127.0.0.1 instead. It's possible `localhost:3306` would also work but I haven't tried that yet.